app/src/main/java/com/framgia/moviedb_31/screen/movieDetail/ItemMovieDetailViewModel.java

```diff
@@ -0,0 +1,32 @@
+package com.framgia.moviedb_31.screen.movieDetail;
+
+import android.databinding.BaseObservable;
+import android.databinding.ObservableField;
+import com.framgia.moviedb_31.data.model.Actor;
+import com.framgia.moviedb_31.data.model.Production;
+
+public class ItemMovieDetailViewModel extends BaseObservable {
+
+    public ObservableField<Production> mProduction = new ObservableField<>();
+    public ObservableField<Actor> mActor = new ObservableField<>();
```

Comment:
khởi tạo mProduction & mActor trong constructer
```suggestion
    public ObservableField<Actor> mActor;
```